src/components/panel/panel.spec.js

```diff
@@ -2823,12 +2837,12 @@ describe('$mdPanel', function() {
     attachedElements.push(element);
   }
 
-  function clickPanelContainer() {
+  function clickPanelContainer(container) {
     if (!panelRef) {
       return;
     }
 
-    var container = panelRef.panelContainer;
+    var container = container || panelRef.panelContainer;
```

Comment:
This should actually read as `container = container || panelRef.panelContainer`. I forgot to remove the `var` before it.